o-fish-ios/Views/PatrolBoat/PatrolBoatView.swift

```diff
@@ -18,6 +18,7 @@ struct PatrolBoatView: View {
     @State private var isActiveRootFromSearchView = false
     @State private var isActiveRootFromShowingPatrolSummaryView = false
     @State private var resetLocation = {}
+    @State private var showingBoardingSubmitted = false
```

Comment:
we can show this alert from any screen, what is the purpose to add this state/bindings arguments flow?